src/tests/dotnet-stack/DotnetStack.UnitTests.csproj

```diff
@@ -1,12 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <ProjectReference Include="../../Tools/dotnet-stack/dotnet-stack.csproj" />
     <ProjectReference Include="../Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.UnitTests.csproj" />
```

Comment:
Same question on this dependency.

Reply:
Changing to UnitTestTargetFrameworks property.

Reply:
I meant why the dependency on Microsoft.Diagnostics.NETCore.Client.UnitTests.csproj. If we shell out, multitargeting the tests doesn't make much sense.

Reply:
Removed it.